src/pe/import.rs

```diff
@@ -83,9 +113,7 @@ impl<'a> ImportLookupTableEntry<'a> {
 }
 
 // get until entry is 0
-pub type ImportAddressTable = Vec<u32>;
-
-pub const SIZEOF_IMPORT_ADDRESS_TABLE_ENTRY: usize = 4;
+pub type ImportAddressTable = Vec<usize>;
```

Comment:
This should be `u64`, and the import address table parsing also needs fixing to read either `u32` or `u64` based on whether it is PE32+.

Reply:
Thank you for the comments. I changed ImportAddressTable to an enum that holds a Vec<u32> or Vec<u64>, and added a method in the Bitfield trait that converts Vec<T> to an ImportAddressTable. Not sure if this is the optimal solution though.

Reply:
For unified 32-bit/64-bit types, I think the rest of the crate tends to convert the fields to `u64`, so I think a plain `Vec<u64>` is better. You can add a method in the Bitfield trait to convert a value to `u64`.